src/main/scala/io/kf/etl/processors/common/converter/EntityConverter.scala

```diff
@@ -88,7 +88,7 @@ object EntityConverter {
       mondo_id_diagnosis = mondoIdDiagnosos,
       source_text_diagnosis = diagnosis.source_text_diagnosis,
       uberon_id_tumor_location = diagnosis.uberon_id_tumor_location,
-      source_text_tumor_location = diagnosis.source_text_tumor_location,
+      source_text_tumor_location = diagnosis.source_text_tumor_location.map(tumor_location => tumor_location.split(";").toSeq).getOrElse(Seq.empty),
```

Comment:
I think it can be symplified as `.map(_.split(";").toSeq).getOrElse(Seq.empty)`.